man/redfishpower.8.in

```diff
@@ -60,6 +60,10 @@ Set path and optional post data to turn on plug.
 .I "setoffpath <path> [postdata]"
 Set path and optional post data to turn off plug.
 .TP
+.I "setplugs plugnames hostindexes"
+Set plug names for index of hosts input to redfishpower.  Can be
```

Comment:
Would it be better to say "associate a plug name with one of the hostnames specified on the command line, referred to by its zero origin index".